### PR TITLE
JavaScript Markdown Scraper

### LINKED ISSUE #74: Make a Javascript scraper
For merging nodes we'd like to be able to automatically load awesome lists
- Make markdown scraper (md_scrape.py) into a Javascript scraper for usage in #75

### PR DESCRIPTION
**Pull request by susan-lin**
### In this PR
Converted the existing markdown scraper from Python into Javascript, adding the following changes
- restructured the creation of a node in the Scope class
- added a level property in the Scope class 
- replaced the original hierarchy comparison function with a comparison between the levels of Scope instances

Added a couple tests.

### Note
- Resolves #74 
- disabled ESLint in some places in the scraper file

### CODE REVIEW

File: v4/src/md_scraper.test.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
+import { graphFromText } from './md_scraper'; // eslint-disable-line
```
Comment (rustielin) [inline v4/src/md_scraper.test.js:1]: maybe move this to dedicated test dir?

Comment (rianadon) [reply v4/src/md_scraper.test.js:1]: For what it's worth I've read that generally [unit tests are best placed next to the file they're testing](https://medium.com/@JeffLombardJr/organizing-tests-in-jest-17fc431ff850). It also helps keeps the import statements sane.

Comment (rustielin) [reply v4/src/md_scraper.test.js:1]: Ah haven't done that before but that does make sense. We can move some of the files/directories around in the future if it gets too bloated

**Comment by rustielin** [comment]
Can we run the tests in travis?

File: v4/src/md_scraper.test.js (hunk 1 above)
Comment (susan-lin) [reply v4/src/md_scraper.test.js:1]: Oh I didn't realize either -- sounds good though!

**Comment by susan-lin** [comment]
@rustielin added (and ran) the test in travis!

**Pull request merged**